Wrappers/Python/ccpi/viewer/CILViewerBase.py

```diff
@@ -394,3 +396,25 @@ def setVisualisationDownsampling(self, value):
 
     def getVisualisationDownsampling(self):
         return self.visualisation_downsampling
+
+    def getCurrentAxisLabelsText(self):
+        '''Returns the current labels on the axis widget.'''
+        om = self.orientation_marker.GetOrientationMarker()
+        return [om.GetXAxisLabelText(), om.GetYAxisLabelText(), om.GetZAxisLabelText()]
+
+    def setAxisLabels(self, labels=['x', 'y', 'z'], overwrite_flag=True):
```

Comment:
why do you need the `overwrite_flag`?

This is also related to the point above, I would say that you don't need the member `axisLabelsText` as you get the labels from the `getCurrentAxisLabelsText` method. Or am I missing something?

Reply:
We need to store the information on the 3D axes because when we change the view in 2D we are deleting one of the axes labels. Does this sound ok?

Reply:
I don't think we need the `overwrite_flag` parameter, it can be implicit.

Reply:
How do you set the labels after two rotations? eg. x and then y? you will have lost the information on the x labelling if you do not save the labels. right?